src/Assetic/Filter/CoffeeScriptFilter.php

```diff
@@ -20,19 +20,17 @@
  * @link http://coffeescript.org/
  * @author Kris Wallsmith <kris.wallsmith@gmail.com>
  */
-class CoffeeScriptFilter extends BaseNodeFilter
+class CoffeeScriptFilter extends BaseProcessFilter
 {
     private $coffeeBin;
-    private $nodeBin;
 
     // coffee options
     private $bare;
     private $noHeader;
 
-    public function __construct($coffeeBin = '/usr/bin/coffee', $nodeBin = null)
+    public function __construct($coffeeBin = 'coffee')
```

Comment:
This actually doesn't break BC. If you're still passing the `$nodeBin` argument, it will simply be ignored by the constructor. So no one should need to change how they are constructing the filter. However I realise that it's probably better to keep the absolute default but drop the second argument (with no side effects).